webapp/src/Routes.js

```diff
@@ -1,17 +1,24 @@
 import React from 'react'
 import { Switch, Route } from 'react-router-dom'
 import Page from './widgets/Page/Page'
-import Home from './features/Home'
+import NurseVisitArchive from './features/NurseVisitArchive'
+import Settings from './features/Settings/Settings'
+import NurseVisitItem from './features/NurseVisitItem'
 export default function Routes() {
 	return (
 		<Switch>
-			<PageRoute path='/'>
-				<Home />
+			<PageRoute path='/settings'>
+				<Settings />
+			</PageRoute>
+			<PageRoute path='/nurse/visit/:id'>
+				<NurseVisitItem />
+			</PageRoute>
+			<PageRoute path='/nurse/visit'>
+				<NurseVisitArchive />
```

Comment:
/nurse/visit to be /visits - (if no other visits like doctors etc.) - if yes, rename  /nurse/visit to  /nurse/visits

Reply:
In future we may have provider/visits also